packages/runtime/src/templates/getPageResolver.ts

```diff
@@ -46,7 +55,7 @@ export const getSinglePageResolver = async ({
 }) => {
   const dependencies = await getDependenciesOfFile(sourceFile)
   // We don't need the actual name, just the relative path.
-  const functionDir = resolve(functionsDir, 'functionName')
+  const functionDir = resolvePosix(functionsDir, 'functionName')
```

Comment:
Any reason for renaming the named export? It's only used here.

```javascript
import { relative, resolve as resolvePosix } from 'pathe'
```

Reply:
I'm also importing the regular `resolve` from `path`. The `pathe` version normalises everything to posix (and more comprehensively than `path.posix.resolve`)